### DISCUSSION TITLE
Disable Buttons on Pump (Via remote)

### DISCUSSION THREAD

**Discussion by nick-hills** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#523] [General]
Is it possible to lock-out the local buttons from the remote webapp / mqtt?

From the docs, I think I  can comment them out before compiling, but id like to be able to control wether or not the buttons are enabled depending on a switch?

**Comment by visualapproach** [comment]
It's not possible for the moment. Can you state the use case?

**Reply by nick-hills** [reply]
Kids!

I'd like to be able to disable the pump controls to stop 'accidental' button presses.

**Reply by visualapproach** [reply]
Got it! I will put it on the todo list. Meanwhile put some schoolwork on the pump. It should keep them from approaching...
